umlet_source/component_instance_2.3.uxf

```diff
@@ -0,0 +1,175 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
For this umlet source as well as the others, why include the "_2.3" prefix? Can we just call these "component_instance" and let git manage keeping track of the versions?